src/mdo/java/WrapperProperties.java

```diff
@@ -350,28 +325,32 @@ public synchronized void loadFromXML(InputStream in) throws IOException, Invalid
 
     @Override
     public void storeToXML(OutputStream os, String comment) throws IOException {
-        Properties props = new Properties();
-        props.putAll(getter.get());
+        OrderedProperties props = new OrderedProperties(getter.get());
         props.storeToXML(os, comment);
     }
 
     @Override
     public void storeToXML(OutputStream os, String comment, String encoding) throws IOException {
-        Properties props = new Properties();
-        props.putAll(getter.get());
+        OrderedProperties props = new OrderedProperties(getter.get());
         props.storeToXML(os, comment, encoding);
     }
 
 
     private Object writeReplace() throws java.io.ObjectStreamException {
-        Properties props = new Properties();
-        props.putAll(getter.get());
+        OrderedProperties props = new OrderedProperties(getter.get());
```

Comment:
please inline, considering sonar.

- https://github.com/pmd/pmd/issues/5770

Reply:
yes it is, its declared `L:340` and used `L:341` in `return props;`.

lets just ask judge `PMD`, im sure it will point out the same.  

<img width="1795" alt="image" src="https://github.com/user-attachments/assets/68ae2c0e-6208-4a7e-9c2f-408589882f9c" />